src/assets/i18n/es.json

```diff
@@ -604,6 +569,289 @@
             "Unit for Table": "Unidad para la Tabla"
         }
     },
+    "invoices": {
+        "Invoices": "Facturas",
+        "New Invoice": "Nueva Factura",
+        "Set Items": "Establecer Elementos",
+        "Customer": "Cliente",
+        "Date": "Fecha",
+        "Status": "Estatus",
+        "Billed": "Cobrada",
+        "Manage": "Administrar",
+        "Open PDF": "Abrir PDF",
+        "Proof of Payment": "Prueba de Pago",
+        "Fix Inventory": "Arreglar Inventario",
+        "Cancel Invoice": "Cancelar Factura",
+        "Search": "Buscar",
+        "Create Export Invoice": "Crear Factura de Exportación",
+        "Create Invoice": "Crear Factura",
+        "Set Invoice Items": "Establecer Elementos de Factura",
+        "Add Item": "Agregar Elemento",
+        "Name": "Nombre",
+        "Price": "Precio",
+        "Affects Inventory": "Afecta el Inventario",
+        "Inventory Info": "Información de Inventario",
+        "Product": "Producto",
+        "Quantity": "Cantidad",
+        "Plant": "Planta",
+        "Tank": "Tanque",
+        "Delete Item": "Borrar Elemento",
+        "Add Inventory Info": "Agregar Información de Inventario",
+        "Delete Inventory Info": "Borrar Información de Inventario",
+        "Phone": "Teléfono",
+        "Street": "Calle",
+        "City": "Ciudad",
+        "State": "Estado",
+        "Country": "País",
+        "Items": "Elementos",
+        "Storage": "Almacenamiento",
+        "Document Info": "Información de Documento",
+        "Printable Document": "Documento Imprimible",
+        "Seller Info": "Información del Vendedor",
+        "Buyer Info": "Información del Comprador",
+        "Item List": "Lista de Elementos",
+        "New Item": "Nuevo Elemento",
+        "Item Name": "Nombre del Elemento",
+        "Add Info": "Agregar Información",
+        "Item Description": "Descripción del Elemento",
+        "Select Tickets": "Seleccione Boletas",
+        "New Export Invoice": "Nuevo Factura de Exportación",
+        "Ticket": "Boleta",
+        "Search Tickets": "Buscar Boletas",
+        "Next": "Próximo",
+        "Confirm Invoice Information": "Confirmar Información de la Factura",
+        "Seller data": "Información del Vendedor",
+        "Address": "Address",
+        "Customer data": "Información del Cliente",
+        "Other": "Otra Información",
+        "Incoterm": "Incoterm",
+        "None": "None",
+        "Delivery at Place": "Delivery at Place",
+        "EX Works": "EX Works",
+        "Free On Board": "Free On Board",
+        "Delivery": "Delivery",
```

Comment:
Are there any good translations for these phrases?

Reply:
No translation, except for "None"

Reply:
Alright, updated it to "Ninguno".